test/e2e-go/features/participation/onlineOfflineParticipation_test.go

```diff
@@ -96,18 +101,28 @@ func waitForAccountToProposeBlock(a *require.Assertions, fixture *fixtures.RestC
 }
 
 func TestNewAccountCanGoOnlineAndParticipate(t *testing.T) {
-	/*if runtime.GOOS == "darwin" {
-		t.Skip()
-	}
 	if testing.Short() {
 		t.Skip()
-	}*/
-	t.Skip() // temporary disable the test since it's failing.
+	}
 
 	t.Parallel()
 	a := require.New(fixtures.SynchronizedTest(t))
 
+	// Make the seed lookback shorter, otherwise will waite for 320 rounds
+	consensus := make(config.ConsensusProtocols)
+	shortPartKeysProtocol := config.Consensus[protocol.ConsensusCurrentVersion]
+	shortPartKeysProtocol.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
+	shortPartKeysProtocol.SeedLookback = 2
+	shortPartKeysProtocol.SeedRefreshInterval = 8
+	if runtime.GOARCH == "amd64" {
+		// amd64 platforms are generally quite capable, so accelerate the round times to make the test run faster.
+		shortPartKeysProtocol.AgreementFilterTimeoutPeriod0 = 1 * time.Second
+		shortPartKeysProtocol.AgreementFilterTimeout = 1 * time.Second
+	}
+	consensus[protocol.ConsensusVersion("shortpartkeysprotocol")] = shortPartKeysProtocol
+	
 	var fixture fixtures.RestClientFixture
+	fixture.SetConsensus(consensus)
```

Comment:
One node network would be enough for running this test.

Reply:
Done.